mcodingbot/utils/peps.py

```diff
@@ -122,3 +122,6 @@ def truncated_title(self) -> str:
 
     def __str__(self) -> str:
         return f"PEP {self.number}: [{self.title}]({self.link})"
+
+    def __hash__(self) -> int:
+        return self.number
```

Comment:
thinking a docstring could help a bit with clarity. May add that.

Reply:
no need imo, the method is simple enough